tasks/main.yml

```diff
@@ -105,7 +123,7 @@
       become_user: "{{ atom_user }}"
       environment:
         - "{{ atom_pool_php_envs }}"
-        - PATH: "{{ ansible_env.PATH }}:/opt/rh/rh-php{{ php_version }}/root/bin/"
+        - PATH: "{{ ansible_env.PATH }}:{{ php_rh_centos_path | default('') }}"
```

Comment:
This task is the only one having `PATH: "{{ ansible_env.PATH }}:{{ php_rh_centos_path | default('') }}"` (the other tasks have something like `PATH: "{{ ansible_env.PATH }}:{{ php_rh_centos_path }}"` or `PATH: "{{ ansible_env.PATH }}:{{ php_rh_centos_path | default('/opt/rh/rh-php{{ php_version }}/root/bin/') }}"`
Because `php_rh_centos_path` is always set when using RedHat or CentOS (in `vars/CentOS.yml` or `vars/RedHat.yml`), and the extra path is not needed in Ubuntu, should the default be empty in all the other tasks as well? (i.e., use `PATH: "{{ ansible_env.PATH }}:{{ php_rh_centos_path | default('') }}"` in all cases)